frontend/src/pages/RunList.tsx

```diff
@@ -503,7 +503,7 @@ class RunList extends React.PureComponent<RunListProps, RunListState> {
         usePlaceholder: false,
         versionId: undefined,
       };
-    } else if (!!RunUtils.getWorkflowManifest(displayRun.run)) {
+    } else if (displayRun.run.pipeline_spec?.pipeline_manifest) {
```

Comment:
This page supports both V1 and V2, let's make sure we check for both `pipeline_manifest` and `workflow_manifest`. As long as any one of them is not empty, you can show the placeholder string.